### PR TITLE
python@3.8 3.8.2

### PR DESCRIPTION
**Pull request by bayandin**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by gengjiawen — APPROVED** [review]

**Comment by bayandin** [comment]
Thanks @Bo98!

File: Formula/python@3.8.rb [hunk 1]
```diff
@@ -41,18 +41,18 @@ class PythonAT38 < Formula
   skip_clean "bin/easy_install3", "bin/easy_install-3.4", "bin/easy_install-3.5", "bin/easy_install-3.6", "bin/easy_install-3.7", "bin/easy_install-3.8"
 
   resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/f7/b6/5b98441b6749ea1db1e41e5e6e7a93cbdd7ffd45e11fe1b22d45884bc777/setuptools-42.0.2.zip"
-    sha256 "c5b372090d7c8709ce79a6a66872a91e518f7d65af97fca78135e1cb10d4b940"
+    url "https://files.pythonhosted.org/packages/68/75/d1d7b7340b9eb6e0388bf95729e63c410b381eb71fe8875cdfd949d8f9ce/setuptools-45.2.0.zip"
+    sha256 "89c6e6011ec2f6d57d43a3f9296c4ef022c2cbf49bab26b407fe67992ae3397f"
```
Comment (bayandin) [inline Formula/python@3.8.rb]:
```suggestion
    url "https://files.pythonhosted.org/packages/df/ed/bea598a87a8f7e21ac5bbf464102077c7102557c07db9ff4e207bd9f7806/setuptools-46.0.0.zip"
    sha256 "2f00f25b780fbfd0787e46891dcccd805b08d007621f24629025f48afef444b5"
```

Comment (Bo98) [reply Formula/python@3.8.rb]: Done

Comment (Bo98) [reply Formula/python@3.8.rb]: Turns out 46.0.0 removed some deprecated features which break a couple builds.

**Pull request closed without merging**